README.md

```diff
@@ -31,45 +31,61 @@ mvn clean package
 **Help**
 
 ```
-java -jar noark-extraction-validator-0.2.0.jar
+java -jar noark-extraction-validator-0.3.0-SNAPSHOT.jar
```

Comment:
I don't think we need to use the SNAPSHOT version. This feature will be released in 0.3.0 so I'd suggest to use `0.3.0` instead of `0.3.0-SNAPSHOT` (to make our lives a little bit easier when doing the release).

Reply:
Good point, fixed.